application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -3301,12 +3301,16 @@ public function get_uploaded_files($sSessionKey, $iSurveyID, $sToken, $responseI
             return ['status' => 'Invalid arguments: both Token and Reponse ID are empty'];
         }
         $criteria = new CDbCriteria();
+        $params = [];
         if (!empty($responseId)) {
-            $criteria->addCondition('id', $responseId);
+            $criteria->addCondition('id = :id');
+            $params[':id'] = $responseId;
         }
         if (!empty($sToken)) {
-            $criteria->addCondition('token', $sToken);
+            $criteria->addCondition('token = :token');
+            $params[':token'] = $sToken;
         }
+        $criteria->params = $params;
```

Comment:
Are you requesting changes just because no params are to be set with `$criteria->compare`?
I understand you prefer it that way, but requesting a change...?